apps/molgenis-components/src/components/tables/AggregateTable.vue

```diff
@@ -145,11 +146,11 @@ export default defineComponent({
         .fetchAggregateData(
           this.tableName,
           {
-            name: this.selectedColumn,
+            name: convertToCamelCase(this.selectedColumn),
```

Comment:
this works. I was thinking if you could call 'column.id' somewhere however then these ids would also show up in select. So this is the right way in this case I agree.